book/src/building-from-source.md

```diff
@@ -35,10 +35,18 @@ RUSTFLAGS="-C target-feature=-crt-static"
 2. Compile from source:
 
    ```sh
+   # Reproducible
    cargo install --path helix-term --locked
    ```
+   ```sh
+   # Optimized
+   cargo +stable install \
```

Comment:
`+stable` is only available if you're using rustup which is not always the case, for example you might use cargo/rustc from your package manager